src/App.js

```diff
@@ -1,16 +1,23 @@
-import { Routes, Route } from "react-router-dom";
-import React from "react";
+import React, { useEffect } from "react";
```

Comment:
useEffect Hook (Fetching data, Reading from local storage, Registering and deregistering listeners)

Reply:
useEffect is used for react life cycles: Mounting, Updating, and Unmounting